### DISCUSSION TITLE
Design Thinking

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#13] [Ideas]
How can we embrace design thinking to transform the way we develop the KFG tool and community? 

Design thinking is an iterative process that focuses on understanding the user, challenging assumptions, and redefining problems to identify alternative strategies and solutions.

5 Key Elements:

1. Human-Centered
2. Creative & Playful
3. Iterative
4. Collaborative
5. Prototype-Driven

**Comment by seekanddefine** [comment]
Step 1. Empathize

-Personas
-Role objectives
-Decisions
-Challenges
-Pain Points

**Reply by avidrucker** [reply]
I have a few people in mind that are needing help to learn things. In each case, there are "gaps" in their knowledge, where they "don't know what they don't know". This strikes me as both as a challenge and as a pain point:
1. How does one become aware that there are gaps?
2. What can a learner do when faced with daunting gaps, without a clear idea of who to ask what questions?

**Comment by seekanddefine** [comment]
Step 2. Define

-Interviews
-Shadowing
-Seek to understand
-Non-judgmental

**Comment by seekanddefine** [comment]
Step 3. Ideate

-Share ideas
-All ideas worthy
-Diverge/Converge
-"Yes and" thinking
-Prioritize

**Comment by seekanddefine** [comment]
Step 4. Prototype

-Mockups
-Storyboards
-Keep it simple
-Fail fast
-Iterate quickly

**Comment by seekanddefine** [comment]
Step 5. Test

-Understand impediments
-What works?
-Role play
-Iterate quickly
